src/components/organisms/Portfolio/index.tsx

```diff
@@ -0,0 +1,83 @@
+import styled from "@emotion/styled";
+import { Grid } from "@mui/material";
+import React, { useState } from "react";
+import Button from "../../atoms/Buttons";
+import Slider from "../Slider";
+import { sliderData } from "../../../data/SliderData";
+import Copy from "../../../assets/icons/copy_portfolio.svg";
+import Trash from "../../../assets/icons/trash_portfolio.svg";
+import EducationDetail from "../EducationDetails";
+import Certifications from "../Certifications";
+
+const Container = styled(Grid)({
+  display: "flex",
+  flexDirection: "column",
+  gap: 32,
+  alignItems: "center",
+  margin: 'auto',
+  width: '60.625rem',
+});
+
+const Buttons = styled(Grid)({
+  position: "relative",
+  marginLeft: "auto",
+  marginBottom: "auto",
+  display: "flex",
+  gap: 12,
+});
+
+const PortfolioIcons = styled(Grid)({
+  display: `flex`,
+  gap: `1.4rem`,
+});
+
+const Portfolio = () => {
+  const [sliderId, setSliderId] = useState<number>(0);
+  const getSliderId = (data: number) => {
+    setSliderId(data);
+  };
+
+  const getSlideData = (id: number) => {
+    switch (id) {
+      case 2:
```

Comment:
Better to use if else here as there are only 2 condition. Switch is useful when there are more than 2 conditions.